app/uk/gov/hmrc/softdrinksindustrylevy/controllers/SdilController.scala

```diff
@@ -59,9 +62,33 @@ class SdilController @Inject()(desSubmissionService: DesSubmissionService,
   }
 
   def checkPendingSubscription(utr: String): Action[AnyContent] = Action.async { implicit request =>
```

Comment:
do we need this function/endpoint any more?

Reply:
I think so, because we call it if there isn't a des sub. We could use the same one however checkPendingSubscription is much lighter as it just talks to the db. Unless you think otherwise?